main.js

```diff
@@ -52,4 +52,64 @@ for (let i = 0; i < arr10.length; i++) {
     if (arr10[i] % 2 === 0) {
         console.log(arr10[i]);
     }
+}
+
+//Lesson 16
+//task 16.5
+let str = 'шалаш';
+let str2 = 'Ирина';
+const  polyndrome = function (poly) {
+    let res = false;
+    for (let i = 0; i <= poly.length; i++) {
+        if (poly[i] !== poly[poly.length - (i + 1)]) {
+            res = false;
+            break;
+        } else
+            res = true;
+    }
+    return res;
+};
+
+console.log(polyndrome(str));
+console.log(polyndrome(str2));
+
+//task 16.6
+let num5 = 5;
+let num10 = 10;
+const min = function (a, b) {
+    if (a > b) {
+        return b;
+    }
+    return a;
+};
+
+console.log(min(num5, num10));
+
+const max = function (a, y) {
+    if (a < y) {
+        return y;
+    }
+    return a;
+};
+
+console.log(max(num5, num10));
+
+const minmax = (num5 > num10) ? num10 : num5;
```

Comment:
Wrong naming. It returns the smaller of the two. Why you call it `...max`?

Reply:
In this case, I decided to show two functions in one, which is why I called it that. But I agree with you, incorrect name. Correcting.